src/phases/query/CollScanComplexPredicate.yml

```diff
@@ -66,6 +79,22 @@ GlobalDefaults:
       ]}
     ]}
 
+  MixedQuery3LevelsDeepWithVeryLargeIn: &MixedQuery3LevelsDeepWithVeryLargeIn
```

Comment:
Is the only difference here from the MixedQuery3LevelsDeep test the addition of the VeryLargeIn clause? If so, maybe add a comment about it? Same goes for below.